docs/stdlib/json.rst

```diff
@@ -132,26 +132,24 @@ possible to cast a JSON value directly into a :eql:type:`tuple`.
         db> select <bool>to_json('true');
         {true}
 
-    Note that a :eql:type:`json` value can be cast into a :eql:type:`str`
-    only when it is a JSON string.  Therefore, while the following will work
-    as expected:
+    A :eql:type:`json` type can also be casted as a :eql:type:`str` type,
+    but only when recognised as a JSON string:
```

Comment:
```suggestion
    only when recognized as a JSON string:
```

We use US English spellings. (I hope to create a style guide for better transparency around these issues.)